include/vfio-user.h

```diff
@@ -228,6 +229,44 @@ struct vfio_user_bitmap_range {
 
 #endif /* VFIO_REGION_TYPE_MIGRATION */
 
+/* Analogous to vfio_device_feature */
+struct vfio_user_device_feature {
+	uint32_t	argsz;
+	uint32_t	flags;
+#define VFIO_DEVICE_FEATURE_MASK	(0xffff)  /* 16-bit feature index */
```

Comment:
If an old kernel is used these won't be defined, see how we handle `VFIO_IOMMU_DIRTY_PAGES_FLAG_START`.